app/views/play/menu/InventoryModal.coffee

```diff
@@ -418,6 +418,7 @@ module.exports = class InventoryModal extends ModalView
 
 
   showItemDetails: (item, showExtra) ->
+    item.heroStatModifier = @selectedHero.getHeroStats()
```

Comment:
Don't really like setting the tightly coupled phantom variable here. A cleaner way to pass the hero stats information to would be to give it to the `ItemDetailsView` in its constructor options, then pass it to the item as a parameter to `getFrontFacingStats`.